web/src/components/newsList.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import loadable from '@loadable/component';
+import { slidesToShowPlugin, arrowsPlugin } from '@brainhubeu/react-carousel';
+
+import NewsCard from './newsCard';
+import MoreButton from './moreButton';
+
+const Carousel = loadable(() => import('./carouselWrapper'));
+
+export default function NewsList({ news }) {
+    const content = {
+        text: 'More ❯',
+        back: '❮ Back',
+    };
+    return (
+        <div className="bg-brand-1 h-full mb-48 pb-48">
+            <h1 className="pt-12 text-center text-white text-4xl uppercase">
+                News
+                <span className="absolute bg-brand-4 h-1 left-[48%] text-brand-4 top-[119%] w-10" />
+            </h1>
+            <div className="mt-32 relative">
+                <Carousel
+                    plugins={[
+                        {
+                            resolve: arrowsPlugin,
+                            options: {
+                                arrowLeft: (
+                                    <MoreButton isLeft text={content.back} />
+                                ),
+                                arrowLeftDisabled: (
+                                    <MoreButton
+                                        text={content.back}
+                                        isLeft
+                                        isDisabled
+                                    />
+                                ),
+                                arrowRight: (
+                                    <MoreButton text={content.text} isRight />
+                                ),
+                                arrowRightDisabled: (
+                                    <MoreButton
+                                        text={content.text}
+                                        isDisabled
+                                        isRight
+                                    />
+                                ),
+                                addArrowClickHandler: true,
+                            },
+                        },
+                        {
+                            resolve: slidesToShowPlugin,
+                            options: {
+                                numberOfSlides: 3,
```

Comment:
What is this value for?

Reply:
how many slides we want in the container upon page render, before it slides @asasmith

Reply:
I'm curious what happens if you omit this. The designs show a 4th slide entering on the right, are you able to match that look at 1280px?

Reply:
> html
> ![screenshot](https://camo.githubusercontent.com/730812e0b7a05faaadd18d663fb0129e9a74799c7ce21f410709aaf292961e1f/68747470733a2f2f6e65746c6966792d636f636f6f6e2e6e65746c6966792e6170702f2e6e65746c6966792f66756e6374696f6e732f66657463683f636f64653d33303726706174683d65794a7a6158526c58326c6b496a6f694d5467314e7a67354f4745744f5468684e533030596a6c684c57466a5a6a63745a6a64684d5441344f545a6a5a545668496977695a4756776247393558326c6b496a6f694e6a4269595467325a6a5a6b4f4749775a4749774d4441344e44673559574a6a49697769615751694f694a6a5a475a684d4449344e4330344d7a466a4c54526a5a6d4574596d5a6d4d433034597a4d314e4451315a4467785a44496966513d3d)
> There's a lot more space between the items than what the designs show. Do we have control over that?
> 
> designs
> ![image](https://camo.githubusercontent.com/8bdf95e53473fc36d36714763dc1919a5696c6e0504f8fb6d97babb5c236f83e/68747470733a2f2f6e65746c6966792d636f636f6f6e2e6e65746c6966792e6170702f2e6e65746c6966792f66756e6374696f6e732f66657463683f636f64653d33303726706174683d65794a7a6158526c58326c6b496a6f694d5467314e7a67354f4745744f5468684e533030596a6c684c57466a5a6a63745a6a64684d5441344f545a6a5a545668496977695a4756776247393558326c6b496a6f694e6a4269595467325a6a5a6b4f4749775a4749774d4441344e44673559574a6a49697769615751694f694a684e4749784d4455355a6931684f5467314c54526d595759744f5441324e7930344d6a42695a5751774e7a4a6a4d32556966513d3d)
> 
> Browser metadata
> [Open Deploy Preview](https://deploy-preview-18--airtec.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60ba86f6d8b0db0008489abc&commentID=60ba89635431c211126cb4b1&resolution=resolved)

So, the problem here is that without passing how many items we want displayed in the container, it'll only show one at a time. its trying to display the amount of cards we have passed to the slidesToShowPlugin object. This might not be avoidable with using this package, but i will see what we can do

Reply:
@asasmith its trying to stretch the cards throughout the container thats why the space between the cards increases as the screen increases.